docker-compose.yml

```diff
@@ -71,6 +72,8 @@ services:
   discovery:
     restart: always
     build: ./external/discovery
+    ports:
+      - "5567:5567"
```

Comment:
I think it's better to only expose the REST port from the api gateway in order to isolate as much as possible the app's environment. Maybe to expose them through the development `docker-compose` file would be better.